src/js/filter.js

```diff
@@ -82,4 +84,6 @@ class Filter {
     this.$sort.addEventListener('change', this.update.bind(this))
     this.$filters.forEach($e => $e.addEventListener('change', this.update.bind(this)))
   }
-}
\ No newline at end of file
+}
+
+export default Filter
```

Comment:
Minor: missing newline at end, was missing before, too.

Reply:
Added it, thanks for noticing!